scripts/scraper.py

```diff
@@ -0,0 +1,81 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from bs4 import BeautifulSoup
```

Comment:
@judeleonard where are you using it?

Reply:
https://github.com/judeleonard/population-city/blob/population_city/scripts/scraper.py#L1

here

Reply:
@judeleonard  you imported `BeautifulSoup` but I couldn't find where are you using it.

Reply:
The beautifulSoup is used here https://github.com/judeleonard/population-city/blob/6f193c14dc59db3af4e6b78971b1202bb1d0fc9b/scripts/scraper.py#L39 as a flavour for reading the HTML table